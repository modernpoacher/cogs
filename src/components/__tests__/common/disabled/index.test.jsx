import React from 'react'
import renderer from 'react-test-renderer'

import Component from '#components/common/disabled'

describe('#components/common/disabled', () => {
  describe('<Component />', () => {
    it('renders', () => {
      return expect(renderer.create(<Component />).toJSON())
        .toMatchSnapshot()
    })
  })
})
