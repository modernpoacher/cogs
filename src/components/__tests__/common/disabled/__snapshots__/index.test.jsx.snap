// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`#components/common/disabled <Component /> renders 1`] = `
<span
  className="is-disabled"
>
  *
</span>
`;
